setup.py

```diff
@@ -93,49 +104,53 @@ def is_requirement(line):
         bool: True if the line is not blank, a comment,
         a URL, or an included file
     """
-    return line and line.strip() and not line.startswith(("-r", "#", "-e", "git+", "-c"))
+    return (
+        line and line.strip() and not line.startswith(("-r", "#", "-e", "git+", "-c"))
+    )
 
 
-VERSION = get_version('shoppingcart', '__init__.py')
+VERSION = get_version("shoppingcart", "__init__.py")
 
-if sys.argv[-1] == 'tag':
+if sys.argv[-1] == "tag":
     print("Tagging the version on github:")
     os.system("git tag -a %s -m 'version %s'" % (VERSION, VERSION))
     os.system("git push --tags")
     sys.exit()
 
-README = open(os.path.join(os.path.dirname(__file__), 'README.md')).read()
-CHANGELOG = open(os.path.join(os.path.dirname(__file__), 'CHANGELOG.rst')).read()
+with open(os.path.join(os.path.dirname(__file__), "README.md"), encoding="utf-8") as f:
+    README = f.read()
+with open(
+    os.path.join(os.path.dirname(__file__), "CHANGELOG.rst"), encoding="utf-8"
+) as f:
+    CHANGELOG = f.read()
 
 setup(
-    name='shoppingcart',
+    name="shoppingcart",
     version=VERSION,
     description="""Provides otherwise deprecated appsembler_api as a plugin LMS app""",
-    long_description=README + '\n\n' + CHANGELOG,
-    author='Appsembler, Inc.',
-    author_email='john@appsembler.com',
-    url='https://github.com/appsembler/legacy-appsembler-api',
+    long_description=README + "\n\n" + CHANGELOG,
+    author="Appsembler, Inc.",
+    author_email="john@appsembler.com",
+    url="https://github.com/appsembler/legacy-appsembler-api",
     packages=[
-        'shoppingcart',
+        "shoppingcart",
     ],
     include_package_data=True,
-    install_requires=load_requirements('requirements/base.in'),
+    install_requires=load_requirements("requirements/base.in"),
     python_requires=">=3.5",
     zip_safe=False,
-    keywords='Python edx',
+    keywords="Python edx",
     classifiers=[
-        'Development Status :: 3 - Alpha',
-        'Framework :: Django',
-        'Framework :: Django :: 2.2',
-        'Intended Audience :: Developers',
-        'License :: Other/Proprietary License',
-        'Natural Language :: English',
-        'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.5',
+        "Development Status :: 3 - Alpha",
+        "Framework :: Django",
+        "Framework :: Django :: 2.2",
```

Comment:
```suggestion
        "Framework :: Django :: 4.2",
```
Sumac is on Django 4.2

Reply:
Ah yeah I haven't touched the metadata in the setup.py. Maybe we can address that later if you think it's important? We're not publishing to pypi, so it's probably fine to leave as-is.